cmd/vic-machine/main.go

```diff
@@ -72,9 +79,21 @@ func main() {
 	} else {
 		app.Version = fmt.Sprintf("%s-%s", BuildID, CommitID)
 	}
+	// Open log file
+	f, err := os.OpenFile(LOG_FILE, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0644)
+	if err != nil {
+		fmt.Printf("Error opening logfile %s: %v", LOG_FILE, err)
+	}
+	defer f.Close()
```

Comment:
This should be inside an `} else {`, otherwise will panic() in the case where we cannot OpenFile() above.